apps/cowswap-frontend/src/common/analytics/types.ts

```diff
@@ -0,0 +1,56 @@
+import { AnalyticsCategory, Category, GtmEvent, toGtmEvent } from '@cowprotocol/analytics'
+
+/**
+ * CowSwap-specific analytics categories
+ * Extends the base Category enum with CowSwap-specific values
+ */
+export enum CowSwapCategory {
```

Comment:
Same suggestion, rename to `AnalyticsCategory` or `CowSwapAnalyticsCategory`

Reply:
Addressed (`CowSwapAnalyticsCategory`)